pages/login.html

```diff
@@ -62,6 +62,11 @@
       margin-bottom: 0;
     }
   </style>
+  <script src="https://browser.sentry-cdn.com/5.6.3/bundle.min.js" integrity="sha384-/Cqa/8kaWn7emdqIBLk3AkFMAHBk0LObErtMhO+hr52CntkaurEnihPmqYj3uJho" crossorigin="anonymous">
+  </script>
+  <script type="text/javascript">
+    Sentry.init({ dsn: 'https://39e3ef4785bd4908838cd0d72d714720@sentry.service.dsd.io/152' });
```

Comment:
Sentry's documentation about "Browser JavaScript" integration tells to set the dsn as you do - clearly they know this is sent to clients (browsers) and they doesn't warn against publishing it so it should be fine.

> a value which we call a DSN, or Data Source Name. It looks a lot like a standard URL, but it’s actually just a representation of the configuration required by the Sentry SDKs. It consists of a few pieces, including the protocol, public key, the server address, and the project identifier.

So according to their documentation it doesn't include any authentication/authorisation tokens or anything sensitive (it includes the public key which is ehm...public) so don't imagine this being a problem 👍 (but obviously I'm not "The Security Guy (TM)" 😁).